apps/studio/lib/configcat.ts

```diff
@@ -20,5 +20,5 @@ function getClient() {
 export async function getFlags(user?: User) {
   return user?.email !== undefined
     ? await getClient().getAllValuesAsync(new configcat.User(user.email))
-    : await getClient().getAllValuesAsync()
+    : await getClient().getAllValuesAsync(new configcat.User(''))
```

Comment:
```js
return await getClient().getAllValuesAsync(new configcat.User(user?.email ?? ''))
```

Reply:
oh right that's way more succinct hahaha